modules/addons/openprovider/resources/views/Support/index.tpl

```diff
@@ -12,7 +12,20 @@
 			<div class="alert alert-danger" role="alert">{$LANG.supportZipExtensionWarning}</div>
 			<input class="btn btn-primary" type="button" value="{$LANG.supportDownloadSupportButton}" disabled>
 		{else}
-			<a class="btn btn-primary" href="{get_route route='supportDownload'}" role="button" target="_blank">{$LANG.supportDownloadSupportButton}</a>
+			<script>
+				fetch("{get_route route='supportDownload'}").then(response => {
+					if (response.status !== 500) {
```

Comment:
can't we check with 200 for a success?

Reply:
I have changed what we have discussed and tested that it still works correctly. 


> can't we check with 200 for a success?